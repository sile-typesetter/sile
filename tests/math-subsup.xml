<sile>
<script src="packages/math" />

Sub-superscripts, text, MathML:
<mathml>
  <mrow>
    <msup> <mi>a</mi> <mi>x</mi> </msup>
    <mo>+</mo>
    <msub> <mi>x</mi> <mn>2</mn> </msub>
    <mo>−</mo>
    <msubsup> <mi>ζ</mi> <mn>0</mn> <mi>2</mi> </msubsup>
    <mo>,</mo>
    <msubsup> <mrow></mrow> <mn>12</mn> <mn>14</mn> </msubsup>
    <mi mathvariant="normal">C</mi>
  </mrow>
</mathml>

Sub-superscripts, display, MathML:
<mathml mode="display">
  <mrow>
    <msup> <mi>a</mi> <mi>x</mi> </msup>
    <mo>+</mo>
    <msub> <mi>x</mi> <mn>2</mn> </msub>
    <mo>−</mo>
    <msubsup> <mi>ζ</mi> <mn>0</mn> <mi>2</mi> </msubsup>
    <mo>,</mo>
    <msubsup> <mrow></mrow> <mn>12</mn> <mn>14</mn> </msubsup>
    <mi mathvariant="normal">C</mi>
  </mrow>
</mathml>

Sub-superscripts, text, TeX:
<math>
  a^x + x_2 − \zeta_0^2, {}_{12}^{14} C
</math>

Sub-superscripts, text, TeX:
<math mode="display">
  a^x + x_2 − \zeta_0^2, {}_{12}^{14} C
</math>

</sile>
