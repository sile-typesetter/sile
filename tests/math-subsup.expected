Set paper size 	595.275597	841.8897729
Begin page
Mx 	49.7638
My 	50.7905
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 88 69 w=15.1367	(Sub)
Mx 	64.9005
T	16 w=3.3691	(-)
Mx 	68.2696
T	86 88 83 72 85 86 70 85 76 83 87 86 w=50.5420	(superscripts)
Mx 	118.8116
T	15 w=2.2900	(,)
Mx 	123.7788
T	87 72 91 87 w=16.5332	(text)
Mx 	140.3120
T	15 w=2.2900	(,)
Mx 	145.2792
T	48 68 87 75 48 47 w=34.4336	(MathML)
Mx 	179.7128
T	29 w=2.2900	(:)
Mx 	184.6800
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2713 w=4.5400	(nil)
Mx 	189.2200
My 	47.6905
Set font 	Libertinus Math;8;400;;normal;;LTR
T	2735 w=4.3280	(nil)
Mx 	196.2351
My 	50.7905
Set font 	Libertinus Math;10;400;;normal;;LTR
T	12 w=6.4800	(nil)
Mx 	204.9722
T	2735 w=5.4100	(nil)
Mx 	209.5722
My 	52.8905
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	215.9794
My 	50.7905
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2061 w=6.4800	(nil)
Mx 	224.7165
T	864 w=4.4100	(nil)
Mx 	229.1265
My 	53.3705
Set font 	Libertinus Math;8;400;;normal;;LTR
T	17 w=3.7200	(nil)
My 	46.9745
T	19 w=3.7200	(nil)
Mx 	233.2765
My 	50.7905
Set font 	Libertinus Math;10;400;;normal;;LTR
T	13 w=2.2000	(nil)
Mx 	237.1431
My 	53.3705
Set font 	Libertinus Math;8;400;;normal;;LTR
T	18 19 w=7.4400	(nil)
My 	46.9745
T	18 21 w=7.4400	(nil)
Mx 	245.0131
My 	50.7905
Set font 	Libertinus Math;10;400;;normal;;LTR
T	36 w=6.4600	(nil)
Mx 	49.7638
My 	62.7905
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 88 69 w=15.1367	(Sub)
Mx 	64.9005
T	16 w=3.3691	(-)
Mx 	68.2696
T	86 88 83 72 85 86 70 85 76 83 87 86 w=50.5420	(superscripts)
Mx 	118.8116
T	15 w=2.2900	(,)
Mx 	123.7851
T	71 76 86 83 79 68 92 w=29.3018	(display)
Mx 	153.0868
T	15 w=2.2900	(,)
Mx 	158.0603
T	48 68 87 75 48 47 w=34.4336	(MathML)
Mx 	192.4939
T	29 w=2.2900	(:)
Mx 	264.3110
My 	84.0099
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2713 w=4.5400	(nil)
Mx 	268.8510
My 	80.2599
Set font 	Libertinus Math;8;400;;normal;;LTR
T	2735 w=4.3280	(nil)
Mx 	275.8312
My 	84.0099
Set font 	Libertinus Math;10;400;;normal;;LTR
T	12 w=6.4800	(nil)
Mx 	284.5335
T	2735 w=5.4100	(nil)
Mx 	289.1335
My 	86.1099
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	295.5057
My 	84.0099
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2061 w=6.4800	(nil)
Mx 	304.2079
T	864 w=4.4100	(nil)
Mx 	308.6179
My 	86.5899
Set font 	Libertinus Math;8;400;;normal;;LTR
T	17 w=3.7200	(nil)
My 	80.1939
T	19 w=3.7200	(nil)
Mx 	312.7679
My 	84.0099
Set font 	Libertinus Math;10;400;;normal;;LTR
T	13 w=2.2000	(nil)
Mx 	316.6346
My 	86.5899
Set font 	Libertinus Math;8;400;;normal;;LTR
T	18 19 w=7.4400	(nil)
My 	80.1939
T	18 21 w=7.4400	(nil)
Mx 	324.5046
My 	84.0099
Set font 	Libertinus Math;10;400;;normal;;LTR
T	36 w=6.4600	(nil)
Mx 	49.7638
My 	105.4480
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 88 69 w=15.1367	(Sub)
Mx 	64.9005
T	16 w=3.3691	(-)
Mx 	68.2696
T	86 88 83 72 85 86 70 85 76 83 87 86 w=50.5420	(superscripts)
Mx 	118.8116
T	15 w=2.2900	(,)
Mx 	123.7809
T	87 72 91 87 w=16.5332	(text)
Mx 	140.3141
T	15 w=2.2900	(,)
Mx 	145.2833
T	55 72 59 w=16.2793	(TeX)
Mx 	161.5626
T	29 w=2.2900	(:)
Mx 	166.5318
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2713 w=4.5400	(nil)
Mx 	171.0718
My 	102.3480
Set font 	Libertinus Math;8;400;;normal;;LTR
T	2735 w=4.3280	(nil)
Mx 	178.0890
My 	105.4480
Set font 	Libertinus Math;10;400;;normal;;LTR
T	12 w=6.4800	(nil)
Mx 	186.8281
T	2735 w=5.4100	(nil)
Mx 	191.4281
My 	107.5480
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	197.8373
My 	105.4480
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2061 w=6.4800	(nil)
Mx 	206.5764
T	864 w=4.4100	(nil)
Mx 	210.9864
My 	108.0280
Set font 	Libertinus Math;8;400;;normal;;LTR
T	17 w=3.7200	(nil)
My 	101.6320
T	19 w=3.7200	(nil)
Mx 	215.1364
My 	105.4480
Set font 	Libertinus Math;10;400;;normal;;LTR
T	13 w=2.2000	(nil)
Mx 	219.0031
My 	108.0280
Set font 	Libertinus Math;8;400;;normal;;LTR
T	18 19 w=7.4400	(nil)
My 	101.6320
T	18 21 w=7.4400	(nil)
Mx 	226.8731
My 	105.4480
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2689 w=6.2100	(nil)
Mx 	49.7638
My 	117.4480
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 88 69 w=15.1367	(Sub)
Mx 	64.9005
T	16 w=3.3691	(-)
Mx 	68.2696
T	86 88 83 72 85 86 70 85 76 83 87 86 w=50.5420	(superscripts)
Mx 	118.8116
T	15 w=2.2900	(,)
Mx 	123.7885
T	87 72 91 87 w=16.5332	(text)
Mx 	140.3217
T	15 w=2.2900	(,)
Mx 	145.2985
T	55 72 59 w=16.2793	(TeX)
Mx 	161.5778
T	29 w=2.2900	(:)
Mx 	264.4360
My 	138.5697
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2713 w=4.5400	(nil)
Mx 	268.9760
My 	134.8197
Set font 	Libertinus Math;8;400;;normal;;LTR
T	2735 w=4.3280	(nil)
Mx 	275.9562
My 	138.5697
Set font 	Libertinus Math;10;400;;normal;;LTR
T	12 w=6.4800	(nil)
Mx 	284.6585
T	2735 w=5.4100	(nil)
Mx 	289.2585
My 	140.6697
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	295.6307
My 	138.5697
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2061 w=6.4800	(nil)
Mx 	304.3329
T	864 w=4.4100	(nil)
Mx 	308.7429
My 	141.1497
Set font 	Libertinus Math;8;400;;normal;;LTR
T	17 w=3.7200	(nil)
My 	134.7537
T	19 w=3.7200	(nil)
Mx 	312.8929
My 	138.5697
Set font 	Libertinus Math;10;400;;normal;;LTR
T	13 w=2.2000	(nil)
Mx 	316.7596
My 	141.1497
Set font 	Libertinus Math;8;400;;normal;;LTR
T	18 19 w=7.4400	(nil)
My 	134.7537
T	18 21 w=7.4400	(nil)
Mx 	324.6296
My 	138.5697
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2689 w=6.2100	(nil)
Mx 	295.2916
My 	780.6177
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20 w=4.6924	(1)
End page
Finish
